NeuroID/src/reactNativeLib/java/com/neuroid/tracker/utils/NIDViewsExtensions.kt

```diff
@@ -31,9 +31,17 @@ fun View.getParents(): String {
 }
 
 private fun getParentsOfView(layers: Int, view: View): String {
-    val childView = view.parent as View
-    return if (layers == 3 || childView.id == android.R.id.content) "" else {
-        "${childView.javaClass.simpleName}/${getParentsOfView(layers + 1, childView)}"
+    if (view.parent == null) {
+        throw Error("Before registerTarget add the view to his parent")
```

Comment:
What is the purpose of this error?

Reply:
good point, removed the check so that we don't throw potentially app crashing exceptions that aren't handled. the cast check will handle the null case as well and in this case we will send back "not_a_view" Thanks.